tools/harness-thci/OpenThread.py

```diff
@@ -3111,6 +3139,14 @@ def role_transition(self, role):
     def setLeaderWeight(self, iWeight=72):
         self.__executeCommand('leaderweight %d' % iWeight)
 
+    @watched
+    def isBorderRoutingEnabled(self):
+        try:
+            self.__executeCommand('br enable')
```

Comment:
The formal way should be checking the version. Better to add comment for the interim trick? 

And better to use some alternative 1.3 specific command which will not change stack( e.g. `br omrprefix` which reads).

Reply:
`br state` should be better.

Reply:
I don't think it's a trick. The network data is changing because of the `BorderRouting` feature. So, checking if `BorderRouting` is enabled should be the suitable solution?

Reply:
> `br state` should be better.

Agree. But there is no `br state` command yet. Maybe we should add it.

Reply:
Updated to use `br omrprefix` in last push.